src/categories/dto/update-category.dto.ts

```diff
@@ -0,0 +1,4 @@
+import { PartialType } from '@nestjs/swagger';
+import { CreateCategoryDto } from './create-category.dto';
+
+export class UpdateCategoryDto extends PartialType(CreateCategoryDto) {}
```

Comment:
I checked up on this PartialType on the doc... it is used with graphql is it really needed?

Reply:
Not only with graphQl 
It’s a typescript built in function that makes all class properties optional